deisctl/deisctl.go

```diff
@@ -41,6 +41,9 @@ Commands, use "deisctl help <command>" to learn more:
   refresh-units     refresh unit files from GitHub
   ssh               open an interacive shell on a machine in the cluster
   help              show the help screen for a command
+  upgrade-prep      prepare a running cluster for upgrade
+  upgrade-takeover  allow an upgrade to gracefully takeover a running cluster
+  rolling-restart   perform a rolling restart for instance units
```

Comment:
The term I cribbed from systemds [docs](http://www.freedesktop.org/software/systemd/man/systemd.unit.html) , its just referring to units created like unit@1.service - so the router, registry and gateway are examples. Its intended to indicate that the restart could only function for units created this way - global or one off units couldn't work.

As for better wording - maybe it should say something more generic? "perform a rolling restart for certain valid Deis components" or something.